docs/apache-airflow/migrations-ref.rst

```diff
@@ -23,7 +23,9 @@ Here's the list of all the Database Migrations that are executed via when you ru
 +--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
 | Revision ID                    | Revises ID       | Airflow Version | Description                                                                           |
 +--------------------------------+------------------+-----------------+---------------------------------------------------------------------------------------+
-| ``be2bfac3da23`` (head)        | ``7b2661a43ba3`` | ``2.2.3``       | Add has_import_errors column to DagModel                                              |
+| ``5e3ec427fdd3`` (head)        | ``be2bfac3da23`` | ``2.2.3``       | Increase length of email and username in ``ab_user`` table to ``256`` characters      |
```

Comment:
Shouldn't we include changing column lengths in `ab_register_user` in this migration as well, given that you update `ab_register_user` table in `models.py`?

Reply:
👍